### DISCUSSION TITLE
Feature to disable trackpad until words complete (laptop users)

### DISCUSSION THREAD

**Discussion by peepo5** [monkeytypegame/monkeytype#2177] [Ideas]
When using monkeytype on my laptop, it is a good experience, but my hands hit the trackpad and it makes the UI elements flash because I tap it (not on purpose) to type fast. It would be nice to have an option where the trackpad is disabled unless a shortcut is pressed or the run is over.

**Comment by Miodec** [comment]
Im not sure if the browser can disable the trackpad completely.
